pkg/apis/serving/v1alpha1/route_validation_test.go

```diff
@@ -74,6 +74,25 @@ func TestRouteValidation(t *testing.T) {
 			},
 		},
 		want: nil,
+	}, {
+		name: "valid annotation",
+		r: &Route{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "valid",
+				Annotations: map[string]string{
+					"networking.knative.dev/disableAutoTLS": "true",
+				},
+			},
+			Spec: RouteSpec{
+				Traffic: []TrafficTarget{{
+					TrafficTarget: v1.TrafficTarget{
+						RevisionName: "foo",
+						Percent:      ptr.Int64(100),
+					},
+				}},
+			},
+		},
+		want: nil,
```

Comment:
this can be elided.

Reply:
most of this file includes it. i'm assuming for clarity since it indicates we don't want any errors. i'll keep it in

Reply:
sure I can swoop and clear all of those myself later :)